src/search/pagefind.jl

```diff
@@ -29,7 +29,7 @@ function build_search_index(root, docs, config, rootpath)
 
     pattern = "*/{$(join(config.index_versions, ","))}/**/*.{html}"
 
-    run(`npx pagefind --site $(root) --glob $(pattern)`)
+    run(`npx pagefind --site $(root) --glob $(pattern) --root-selector article --exclude-selectors pre`)
```

Comment:
Wouldn't that make it unable to search code?  Or do we have different semantics for Documenter documents?

Reply:
Yep. Just giving that a go to compare search-index quality (mostly because I thought that's how flexsearch was configured, but turns out it happily indexes code blocks afaict).